app/ui/client/views/app/room.js

```diff
@@ -492,7 +492,7 @@ Template.room.helpers({
 			return true;
 		}
 
-		return !state.get('subscribed');
+		return !!state.get('subscribed');
```

Comment:
```suggestion
		return state.get('subscribed');
```
I don't think the double negation is needed because `subscribed` is set by doubly negating the `subscription` value: it's always a boolean value.